libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.spec.ts

```diff
@@ -73,11 +76,13 @@ describe('Text editor', () => {
         CommonModule,
         FormsModule,
         ReactiveFormsModule,
+        SkyTextEditorResourcesModule,
         SkyTextEditorModule,
         RouterTestingModule,
       ],
       declarations: [componentType],
       providers: [
+        { provide: SkyAppResourcesService, useClass: SkyLibResourcesService },
```

Comment:
I'm not seeing where this is required. Is it necessary?

Reply:
Somehow I missed this in my review. I removed it.